cypress/e2e/tests/migration/task-manager/sorting.filter.pagination.test.ts

```diff
@@ -62,6 +63,7 @@ describe(["@tier3"], "Filtering, sorting and pagination in Task Manager Page", f
                 applicationsList.forEach((application) => application.create());
             });
         });
+
```

Comment:
extra space

Reply:
Thanks @kpunwatk
Done in https://github.com/konveyor/tackle-ui-tests/pull/1265/commits/d63f38cad5d7b92902200d78b4f986cc9bd9b4d7